apps/api/src/domain/models/term/commands/create-term/create-term.command.ts

```diff
@@ -0,0 +1,70 @@
+import { AggregateType, ICommandBase, LanguageCode } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString, UUID } from '@coscrad/data-types';
+import { CREATE_TERM } from './constants';
+
+export class TermCompositeIdentifier {
+    /**
+     * This is a bit of a hack. It circumvents our `CoscradDataTypes` and may
+     * cause problems for
+     * - Schema management
+     * - Anyone using our API directly (not via front-end)
+     *
+     * The simple answer is that you always have to tack on an
+     * `aggregateCompositeIdentifier`.
+     */
+
+    @NonEmptyString({
```

Comment:
I'm surprised this works. At any rate, I've made it stricter on the next branch.